src/Referent/ManagedAreaUtils.php

```diff
@@ -15,4 +16,13 @@ public static function getCodeFromCommittee(Committee $committee): ?string
 
         return static::getCodeFromCountry($committee->getCountry());
     }
+
+    public static function getCodeFromEvent(Event $event): ?string
```

Comment:
Just a reminder comment to refactor these thow method in another PR in the future. :candy: